src/main/java/sqltoregex/deparser/CreateTableDeParserForRegEx.java

```diff
@@ -271,9 +271,33 @@ private String deParseColumnDefinition(ColumnDefinition columnDefinition) {
         List<String> synsWithSpellingMistake = new LinkedList<>();
 
         for (String keywordSyn : StringSynonymGenerator.generateAsListOrDefault(this.datatypeSynonymGenerator, columnDefinition.getColDataType().toString())){
+            StringBuilder tempColumnDefinition = new StringBuilder();
             if (keywordSyn.matches(".*\\([\\w,\\s]+\\)")){
-                keywordSyn = keywordSyn.replace("(", "\\(");
-                keywordSyn = keywordSyn.replace(")", "\\)");
+                String[] splittedColumnDefinition = keywordSyn.split(" ");
+                Iterator<String> stringIterator = Arrays.stream(splittedColumnDefinition).iterator();
+                while(stringIterator.hasNext()){
+                    String splitElement = stringIterator.next();
+
+                    if(splitElement.contains("(") && splitElement.contains(")")){
+                        tempColumnDefinition.append("\\(");
+                        tempColumnDefinition.append(
+                                SpellingMistake.useOrDefault(
+                                        this.keywordSpellingMistake,
+                                        splitElement.replace("(", "").replace(")","")
```

Comment:
In which cases an semicolon exists? 
Where is this added again in the following LoC

Reply:
i didnt see what u mean or want you wanna change.. there a no semicolons or commas :D

Reply:
nevermind I'm blind light was to bright on the laptop and I confused the parameter comma ._.